docs/Kb.md

```diff
@@ -0,0 +1,14 @@
+This file documents the places the `Kb` (Knowledgebase) are used within ZAP and zap-extensions. (`org.parosproxy.paros.core.scanner.Kb`)
+
+Use of `Kb` is currently exposed through `AbstractPlugin` and `HostProcess` `getKb()` methods.
+
+## URI Level
+
+- `org.zaproxy.zap.extension.ascanrules.SqlInjectionScanRule` tracks
+    - sql/<rdbms_name>
+- `org.zaproxy.addon.commonlib.AbstractAppFilePlugin` scan rules track completed 'tests' to prevent duplication.
```

Comment:
I don't think this should be documented, it's not meant to be used by anyone/anything other than the rules themselves and it's a workaround.

Reply:
Okay, I’ll remove that bit.